Documentation/doc/Documentation/packages.txt

```diff
@@ -2,15 +2,6 @@
 
 \page packages Package Overview
 
-\htmlonly
-<style type="text/css">
-h1 {
-	font-size: 150%;
-    background-color: #EEEDF2;
-}
-</style>
-\endhtmlonly
-
```

Comment:
I think it would be easier to have a big `<div style="…">` containing all the content of the file, to avoid having to hard-code styling in the post-processing.

Reply:
I'll give it a try.

Reply:
When having `<div style="...` the result would be valid for the entire file (so in this case complete gray background).
A possibility would be to wrap each `<h1>`  into a `<div style="...`, this is quite feasible (with an ALIAS) but has at the moment 1 small problem:
 `<div` would set the font size top 150% and the `<h1>` would set it back to 180%
when it is acceptable to have 180% for these headings (I think this would not be a problem) I can implement it.

The example with the 180%:
![example180](https://user-images.githubusercontent.com/5223533/53587964-09f4a900-3b8c-11e9-977b-aadbbb29478c.jpg)

The example with the 150%:
![example150](https://user-images.githubusercontent.com/5223533/53587970-0d883000-3b8c-11e9-8f37-23d4b67d2da3.jpg)